cs3/tx/v1beta1/resources.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018-2019 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+syntax = "proto3";
+
+package cs3.tx.v1beta1;
```

Comment:
Is there a reason for using tx? To a new user, it won't make much sense

Reply:
TX is sort of 'legacy'. The 'original name' was datatx but 'data' is not accepted in the namespace (https://github.com/cs3org/cs3apis/pull/81#issue-455594346). How does 'transfer' sound?

Reply:
Oh okay. I think we can stick with this only then.